### ISSUE TITLE
New Chase Bank Account Site Doesn't Work

### ISSUE THREAD

**Issue by l8rcien** [gorhill/uBlock#1816]
Read first: <https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md>

### Describe the issue
Looks like uBlock has completely broken the new Chase bank account site. With uBlockO addon(in Firefox), extension(in Chrome) enabled, I get "Please turn on javascript in your browser". Completely whitelist Chase site doesn't work, only solution is to disable the addon/extension altogether. 

### One or more specific URLs where the issue occurs
When accessing Chase accounts through https://www.chase.com, after typing in username/password, the page gets redirected to https://www.chase.com/digital/resources/js-disabled

[URL(s) for issue on a specific site are **mandatory**]
https://www.chase.com/digital/resources/js-disabled

### Screenshot in which the issue can be seen
[img]http://i.imgur.com/wA3klSp.jpg[/img]

[Screenshot(s) for visual issues are **mandatory**]
[img]http://i.imgur.com/wA3klSp.jpg[/img]

### Steps for anyone to reproduce the issue

This can only be reproduced by those with Chase accounts where their new site has activated.
1. go to https://www.chase.com
2. enter your username/password
3. if chase has activated your account for their new banking site design, you will then see this error message.

### Your settings

Windows 8.1 Pro

- Browser/version: 
Firefox v47.0.1
Chrome v51.0.2704.106m(64-bit)
- uBlock Origin version: v1.7.6

##### Your filter lists
    default + anti adblock killer reek + all 4 multipurpose filters

**Comment by gorhill** [comment]
Closing and marking as _invalid_, as per [CONTRIBUTING](https://github.com/gorhill/uBlock/blob/master/CONTRIBUTING.md).

**Comment by gorhill** [comment]
> Completely whitelist Chase site doesn't work

You screenshots contradict that you whitelisted `chase.com`. Add `chase.com` to the _Whitelist_ pane in the dashboard.

**Comment by l8rcien** [comment]
Sorry about the screenshot. Here's the one where I whitelisted Chase.com [img]http://i.imgur.com/W28vX5k.jpg[/img]. The only way to make the site work is to disable the addon.

**Comment by gorhill** [comment]
I don't have an account, but I entered random user id and pasword, and I was not shown this page, even with uBO enabled.

**Comment by l8rcien** [comment]
Chase has not rolled out the new site to all account holders. when you enter a random ID/Pass, it will just take you to their regular log-in page; or they will redirect to you their old account management page which works perfectly fine with uBlockO. I have tried bypass cache/manually clear cache but none work. Right now, my workaround is just to manually disable uBlock when I need to access the new site. But I expect more people to experience the same problem with their new site as they roll out the site to all account holders.

**Comment by l8rcien** [comment]
Okay, I figured it out. I looked at the chase page elements, my problem was after the main log-on screen, it briefly got redirected to a sub chase domain, that sub domain was not whitelisted so it got redirected to that error page. It was so brief that I didn't catch the fact uBlockO was active when it passed sub domain. I've now whitelisted chase.com instead of www.chase.com and everything appears to be working fine.

Sorry about the false alarm and thank you for a great adblocker!
